Dockerfile

```diff
@@ -17,15 +17,15 @@ COPY assets/dropboxd.conf /etc/supervisor/conf.d/dropboxd.conf
 # - https://www.dropbox.com/install?os=lnx
 RUN wget -O /tmp/dropbox.tgz            \
          -q https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x${ARCH}-${DROPBOX_VERSION}.tar.gz \
- && tar -zxf /tmp/dropbox.tgz -C /root/ \
+ && tar -zxf /tmp/dropbox.tgz -C /dropbox/ \
  && rm -f /tmp/dropbox.tgz
 
 # download the Dropbox python management script
 RUN wget -O /root/dropbox.py \
          -q http://www.dropbox.com/download?dl=packages/dropbox.py
 
-# move .dropbox-dist directory to /dropbox
-RUN mv -f /root/.dropbox-dist /dropbox
+# rename directory to .dropbox-dist for better maintainence
+RUN mv /dropbox/dropbox-lnx.x${ARCH}-${DROPBOX_VERSION} /dropbox/.dropbox-dist
```

Comment:
i have not been able to duplicate this. manually running through the installation steps, the `.dropbox-dist` directory name appears to be the same as before.

```
root@27ae6d67f817:/# wget -O /tmp/dropbox.tgz -q -q https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86_64-53.4.67.tar.gz

root@27ae6d67f817:/# tar -zxf /tmp/dropbox.tgz -C /root/

root@27ae6d67f817:~# ls -la /root/
total 20
drwx------  3 root root 4096 Jul 20 15:52 .
drwxr-xr-x 33 root root 4096 Jul 20 15:52 ..
-rw-r--r--  1 root root  570 Jan 31  2010 .bashrc
drwxr-xr-x  3 1000 1000 4096 Jul 13 01:57 .dropbox-dist
-rw-r--r--  1 root root  148 Aug 17  2015 .profile
```